src/main/java/seedu/address/model/TutorsPet.java

```diff
@@ -139,6 +139,15 @@ public void removeModuleClass(ModuleClass key) {
         moduleClasses.remove(key);
     }
 
+    /**
+     * Removes all {@code Student}s from the application.
+     * Also removes all {@code UUID}s from each {@code ModuleClass}.
+     */
```

Comment:
Perhaps student manager would be a better name?
```suggestion
    /**
     * Removes all {@code Student}s from the student manager.
     * Also removes all {@code UUID}s from each {@code ModuleClass}.
     */
```

Reply:
```suggestion
    /**
     * Removes all {@code Student}s from the student manager.
     * Also removes all {@code Student UUID}s from each {@code ModuleClass}.
     */
```